authorization-common-api/src/main/java/com/sequenceiq/authorization/annotation/FilterListBasedOnPermissions.java

```diff
@@ -5,13 +5,7 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import com.sequenceiq.authorization.resource.AuthorizationResourceAction;
-
-
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.METHOD)
 public @interface FilterListBasedOnPermissions {
-    AuthorizationResourceAction action();
```

Comment:
Yes, this a good idea to remove these properties since people may think they just need to put up a few annotations and everything will work, which is not the case here.

Reply:
exactly